src/store/state/index.js

```diff
@@ -128,10 +128,10 @@ export default {
      * set up globally in the entire aepp.
      */
     account: {
-      address: null,
+      address: process.env.VUE_APP_ACC_PUBLIC_KEY,
```

Comment:
You've missed the update here `VUE_APP_ACCOUNT_PUBLIC_KEY`

Reply:
it should be done now

Reply:
@maptuhec you've missed this one.